src/crimson/osd/ops_executer.cc

```diff
@@ -704,20 +704,21 @@ void OpsExecuter::fill_op_params_bump_pg_version()
   osd_op_params->pg_trim_to = pg->get_pg_trim_to();
   osd_op_params->min_last_complete_ondisk = pg->get_min_last_complete_ondisk();
   osd_op_params->last_complete = pg->get_info().last_complete;
-  if (user_modify) {
-    osd_op_params->user_at_version = osd_op_params->at_version.version;
-  }
 }
 
 std::vector<pg_log_entry_t> OpsExecuter::prepare_transaction(
   const std::vector<OSDOp>& ops)
 {
   std::vector<pg_log_entry_t> log_entries;
-  log_entries.emplace_back(obc->obs.exists ?
+  log_entries.emplace_back(
+    obc->obs.exists ?
       pg_log_entry_t::MODIFY : pg_log_entry_t::DELETE,
-    obc->obs.oi.soid, osd_op_params->at_version, obc->obs.oi.version,
+    obc->obs.oi.soid,
+    osd_op_params->at_version,
+    obc->obs.oi.version,
     osd_op_params->user_modify ? osd_op_params->at_version.version : 0,
-    osd_op_params->req_id, osd_op_params->mtime,
+    osd_op_params->req_id,
+    osd_op_params->mtime,
```

Comment:
These three hunks seem to be whitespace only?

Reply:
Yes, I've moved them to separate lines for the sake readability (one line, one arg).